core/entrypoint/docker-entrypoint.sh

```diff
@@ -1,33 +1,26 @@
 #!/bin/bash
 
 set -e
+trap 'echo "docker-entrypoint.sh : Error occurred on line $LINENO, exiting."; exit 1;' ERR
 
+# Set default CHORUS_USER if not provided
 if [ -z "$CHORUS_USER" ]; then
     echo "CHORUS_USER is not set. Defaulting to 'chorus'"
     export CHORUS_USER="chorus"
 fi
 
-if /usr/bin/find "/docker-entrypoint.d/" -mindepth 1 -maxdepth 1 -type f -print -quit 2>/dev/null | read v; then
+# Check if /docker-entrypoint.d/ contains any files
+if /usr/bin/find "/docker-entrypoint.d/" -mindepth 1 -maxdepth 1 -type f -print -quit 2>/dev/null | read -r _; then
     echo "$0: /docker-entrypoint.d/ is not empty, will attempt to perform configuration"
 
     echo "$0: Looking for shell scripts in /docker-entrypoint.d/"
-    find "/docker-entrypoint.d/" -follow -type f -print | sort -V | while read -r f; do
+    find "/docker-entrypoint.d/" -follow -type f -print | sort -V | while IFS= read -r f; do
         case "$f" in
-            *.envsh)
+            *.envsh|*.sh)
```

Comment:
they aren't _executed_ in the same fashion, hence you cannot group them. `. $f` vs `$f`

Reply:
Dang me, i got chatgpt'ed when i was looking at how to handle error catching in bash , will be more careful next time ...